irr/src/CIrrDeviceSDL.cpp

```diff
@@ -713,7 +713,7 @@ bool CIrrDeviceSDL::run()
 
 	while (!Close && wrap_PollEvent(&SDL_event)) {
 		// os::Printer::log("event: ", core::stringc((int)SDL_event.type).c_str(),   ELL_INFORMATION);	// just for debugging
-		memset(&irrevent, 0, sizeof(irrevent));
+		irrevent = {};
```

Comment:
I prefer this as I'm not entirely sure about the syntax
```suggestion
		irrevent = SEvent();
```

Reply:
The syntax used by grorp is documented here: https://en.cppreference.com/w/cpp/language/aggregate_initialization.